.github/workflows/playwright.yml

```diff
@@ -0,0 +1,29 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+    - name: Install dependencies
+      run: npm ci
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+    - name: Run Playwright tests
+      run: npm run test:e2e
+    - uses: actions/upload-artifact@v3
+      if: always()
```

Comment:
this was from default playwright init.. i'm cool with removing it ¯\\_(ツ)\_/¯

Reply:
oh, this is saying to run it even if test:e2e fails, so it attaches playwright report to CI file.. will leave this, because it allows us to view things like https://github.com/ipfs/helia-http-gateway/suites/17553401925/artifacts/1002822391